### PR TITLE
Initial conversion to MVVM

### PR DESCRIPTION
**Pull request by mungaiandela**

### CODE REVIEW

File: Source/Core/ViewModel/SignInViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    
+    // MARK: Public Instance Properties
+    
+    public let domain: String
+    
+    // MARK: Public Initializers
+    
+    public init() {
+        
+        self.domain = "andela.com"
```
Comment (Muthama-Kahohi) [inline Source/Core/ViewModel/SignInViewModel.swift]: I think the domain can go to the localized file and be referenced from there

**Comment by youknowitsbarney** [comment]
Rebase for merging.

File: Source/Core/ViewModel/SignInViewModel.swift (hunk 1 above)
Comment (mungaiandela) [reply Source/Core/ViewModel/SignInViewModel.swift]: Does this file exist yet?

Comment (mungaiandela) [reply Source/Core/ViewModel/SignInViewModel.swift]: Never mind found it

Comment (mungaiandela) [reply Source/Core/ViewModel/SignInViewModel.swift]: What is the purpose of this anyway? It's not like andela.com will be a different domain name in say French or German

**Pull request merged**